cpanfile

```diff
@@ -4,6 +4,7 @@ requires 'HTTP::Tiny';
 requires 'IO::Compress::Gzip';
 requires 'URI::Escape';
 requires 'Config::IniFiles';
+requires 'List::Util', '>= 1.45';
```

Comment:
This should be in the feature 'xref_mapping' section, shouldn't it.

Reply:
not sure... it's part of the xref parsing and not actually the mapping...

Reply:
True but the directory which holds all the xref code is called _xref_mapping_ even though it does more than just mapping, moreover that way it is clear that this is not a dependency of the core API and thus will be safe to remove once the repository split has been completed.

Reply:
done